embassy-stm32/src/adc/v2.rs

```diff
@@ -80,15 +80,6 @@ impl super::sealed::InternalChannel<ADC1> for Temperature {
 }
 
 impl Temperature {
-    /// Converts temperature sensor reading in millivolts to degrees celcius
-    pub fn to_celcius(sample_mv: u16) -> f32 {
-        // From 6.3.22 Temperature sensor characteristics
-        const V25: i32 = 760; // mV
-        const AVG_SLOPE: f32 = 2.5; // mV/C
```

Comment:
Maybe we could still have these as associated constant on types? Same with voltage on `VrefInt`.

Although, it has the problem of what type to use. I.e. should AVG_SLOPE be 2500 uV/C u16 or 2.5 mV/C f32.

What do you think?

Reply:
I eventually would like to include these constants for all chips in the hal or the metapac, but as far as I can tell, the information is only available in the datasheets. The values aren't necessarily the same even between chips in same family, and I'm not prepared to look through all 159 PDFs to find out :fearful:

Reply:
Fair point 👍